examples/PowerElectronics/DistributedGeneratorTest/DGTest.cpp

```diff
@@ -74,11 +74,20 @@ int main(int argc, char const* argv[])
                                3.337988298081817e+03,
                                2.684419146397466e+03};
 
-  std::cout << "Test the Relative Error\n";
+  std::cout << "Testing the DistributedGenerator model ...\n";
+  double error_allowed = 10 * std::numeric_limits<double>::epsilon();
   for (size_t i = 0; i < true_vec.size(); i++)
```

Comment:
Wouldn't using `GridKit::Testing::isEqual` on `std::vector` work here? Then, we can just querry `dg->getResidual()` once.